### PR TITLE
Speed element

### PR DESCRIPTION
**Pull request by g4s8**
#353 - added speed element to `people.xml`

### CODE REVIEW

**Comment by g4s8** [comment]
@0crat assign @carlosmiranda

**Comment by 0crat** [comment]
Job [#354](https://github.com/zerocracy/datum/issues/354) is now in scope, role is `REV`

**Comment by 0crat** [comment]
> @0crat assign @carlosmiranda  ([here](https://github.com/zerocracy/datum/issues/354#issuecomment-398398486))

@g4s8 This pull request [#354](https://github.com/zerocracy/datum/issues/354) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1701334055); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

File: xsd/pmo/people.xsd [hunk 1]
```diff
@@ -51,6 +51,7 @@ SOFTWARE.
       <xs:element name="reputation" type="xs:integer"/>
       <xs:element name="jobs" type="xs:integer"/>
       <xs:element name="projects" type="xs:integer"/>
+      <xs:element name="speed" type="xs:integer"/>
```
Comment (carlosmiranda) [inline xsd/pmo/people.xsd]: @g4s8 should be `xs:decimal` type, let's restrict it to only positive values too

File: xml/pmo/people/simple.xml [hunk 2]
```diff
@@ -21,6 +21,7 @@ SOFTWARE.
     <jobs>12</jobs>
     <projects>256</projects>
     <rate>£30.50</rate>
+    <speed>1024</speed>
```
Comment (carlosmiranda) [inline xml/pmo/people/simple.xml]: @g4s8 let's show at least a tenths increment here

File: xml/pmo/speed/-no-speed.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<people version="1" updated="2016-12-29T09:03:21.684Z">
```
Comment (carlosmiranda) [inline xml/pmo/speed/-no-speed.xml]: @g4s8 let's add test for invalid speed too

**Comment by g4s8** [comment]
@carlosmiranda fixed, please look

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@2a5ea468-7e77-489c-80d0-c8be03a5d7e1, thread: AsyncFlush-8
com.zerocracy.farm.sync.SyncProject[77] java.lang.IllegalStateException: Failed to acquire "verbosity/carlosmiranda.xml" in "PMO" in 1min: 61df0a5983f1/3min/0/3/true/false by QuartzScheduler_Worker-1
```

[0.22.7](https://github.com/zerocracy/farm): CID: [1711004597](https://www.0crat.com/C3NDPUA8L/1711004597), Type: "Close job", Author: "rultor"

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#354` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#354](https://github.com/zerocracy/datum/issues/354) is now out of scope

**Comment by carlosmiranda** [comment]
@0crat status

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@62c94ee0-47b0-4040-a917-bc85e86ebac6, thread: pool-24-thread-1
com.zerocracy.farm.sync.SyncProject[77] java.lang.IllegalStateException: Failed to acquire "people.xml" in "PMO" in 2min: e52044b7d4bb/3min/0/3/true/false by QuartzScheduler_Worker-8
```

[0.22.21](https://github.com/zerocracy/farm): Issue: zerocracy/datum#354, Comment: 398942965

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@62c94ee0-47b0-4040-a917-bc85e86ebac6, thread: pool-24-thread-1
com.zerocracy.radars.github.ReSafe[82] java.io.IOException: java.lang.IllegalStateException: Failed to acquire "people.xml" in "PMO" in 2min: e52044b7d4bb/3min/0/3/true/false by QuartzScheduler_Worker-8
com.zerocracy.farm.sync.SyncProject[77] java.lang.IllegalStateException: Failed to acquire "people.xml" in "PMO" in 2min: e52044b7d4bb/3min/0/3/true/false by QuartzScheduler_Worker-8
```

[0.22.21](https://github.com/zerocracy/farm): Issue: zerocracy/datum#354, Action: `created`

**Comment by carlosmiranda** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/datum/issues/354#issuecomment-399050173))

@carlosmiranda This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#354](https://github.com/zerocracy/datum/issues/354) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 23hr
  * @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) will receive 15 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Fdatum%23354%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by ypshenychka** [comment]
@0crat quality good

**Pull request merged**